Sources/GarageStorage/GarageStorage.docc/WorkingWithObjectiveC.md

```diff
@@ -65,31 +69,31 @@ Once you have set the properties to map, if it is a top-level object, you should
     }
 ```
 
-Under the hood, the object's properties gets serialized to JSON, and the types of properties supported in Objective-C are limited, so don't try to park any tricky properties, without first wrapping them in MappableObject. The types supported include:
+Under the hood, the object's properties gets serialized to JSON, and the types of properties supported in Objective-C are limited, so don't try to park any tricky properties, without first wrapping them in a `MappableObject`. The types supported include:
 * Strings (`NSString`)
 * Numbers (both `NSNumber` and associated primitives such as `int`, `double`, and `bool`)
 * Dates (`NSDate`)
-* Dictionaries (`NSDictionary`) where keys are Strings, and values are among the supported types
+* Dictionaries (`NSDictionary`) where keys are NSStrings, and values are among the supported types
 * Arrays (`NSArray`) of the supported types
-* Objects conforming to `MappableObject`
+* Objects conforming to `GSMappableObject`
 
 ### Parking, Retrieving, and Deleting Objects
 
 The Garage methods for parking, retrieving and deleting MappableObject objects are the same as for Swift Codable, with the suffix `Object` added:
 
-* The `MappableObject` equivalent to `park<T>(_)` and `parkAll<T>(_)` are `parkObject(_)` and `parkAllObjects(_)`.
-* The `MappableObject` equivalent to `retrieve<T>(_)` and `retrieveAll<T>(_)` are `retrieveObject(_:identifier:)` and `retrieveAllObjects(_)`.
-* The `MappableObject` equivalent to `delete<T>(_)` and `deleteAll<T>(_)` are `deleteObject(_)` and `deleteAllObjects(_)`.
-* The `MappableObject` equivalent to `retrieve<T>(withStatus:)` is `retrieveObjects(withStatus:)`.
+* `parkObject(_)` and `parkAllObjects(_)`
+* `retrieveObject(_:identifier:)` and `retrieveAllObjects(_)`
+* `deleteObject(_)` and `deleteAllObjects(_)`
+* `retrieveObjects(withStatus:)`
 
 ### Working with an identifier for top-level unique objects
 
-To specify the identifier, conform to `MappableObject` and assign the `ObjectMapping identifyingAttribute`. A `MappableObject` without an `identifyingAttribute` will otherwise be anonymous (a dependent object of a root reference object, or an object in an array). For more details, see *Identifying Attributes* in [Getting Started](GettingStarted.md).
+To specify the identifier, conform to `MappableObject` and assign the `ObjectMapping identifyingAttribute`. A `MappableObject` without an ``identifyingAttribute`` will otherwise be anonymous (an embedded object in a top-level object, or an object in an array). For more details, see *Identifying Attributes* in <doc:GettingStarted>.
```

Comment:
I'm getting a warning about how `identifyingAttribute` doesn't exist here. But I notice that you have it wrapped in two layers of backticks. I think that you want only one, like you have for other functions in lines 84-87, above.